domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/CameraXCameraUseCase.kt

```diff
@@ -369,19 +377,8 @@ constructor(
     }
 
     private fun createPreviewUseCase(): Preview {
-        val availableCameraInfo = cameraProvider.availableCameraInfos
-        val cameraSelector = if (isFrontFacing) {
-            CameraSelector.DEFAULT_FRONT_CAMERA
-        } else {
-            CameraSelector.DEFAULT_BACK_CAMERA
-        }
-        val isPreviewStabilizationSupported =
```

Comment:
Why aren't we checking preview stabilization separately from video stabilization? These two stabilization modes should account for:
Preview -> Stabilization ON
Video -> Stabilization HIGH QUALITY